CONVENTIONS.md

```diff
@@ -156,6 +156,13 @@ In future, when we include the descheduler into OpenShift by default, it will pe
     * E.g. the upgrade of a network plugin must serve pod traffic without disruption (although tiny increases in latency are allowed)
     * All components that currently disrupt end-user workloads must prioritize addressing those issues, and new components may not be introduced that add disruption
 
+#### Priority Classes
```

Comment:
Can you link to a doc defining "priority class" and showing how to set it?

Reply:
It will also help if you mention in the commit message or inline why this is needed i.e. the situation that called for introducing this.

Reply:
I added link to set priority class and add a scenario that made us enforce this convention.